src/ent/bit_rack.h

```diff
@@ -1,12 +1,19 @@
 #ifndef BIT_RACK_H
 #define BIT_RACK_H
 
+#if defined(__APPLE__)
+#include "../../compat/endian.h"
+#else
+#include <endian.h>
+#endif
+
```

Comment:
We now handle this in compat/endian_conv.h instead of checking the OS in a bunch of different files. If new endian function are needed you can add them to compat/endian_conv.h

Reply:
Removed this, only including endian_conv.h